deeplearning4j-core/src/main/java/org/deeplearning4j/nn/layers/feedforward/rbm/RBM.java

```diff
@@ -316,11 +316,11 @@ public Layer transpose() {
      * @param v the visible layer
      * @return the approximated activations of the visible layer
      */
-    public INDArray propUp(INDArray v,boolean training) {
+    public INDArray propUp(INDArray v, boolean training) {
         INDArray W = getParam(PretrainParamInitializer.WEIGHT_KEY);
         if(training) {
-            if(conf.isUseDropConnect() && training) {
-                if (conf.getLayer().getDropOut() > 0 && training) {
```

Comment:
Change is logically correct, but the 3 nested if statements there isn't great.
Can just be "if(training && conf.isUseDropConnect() && conf.getLayer().getDropOut())" instead of if(){ if(){ if(){ ...}}}